src/adapters/postgres/user-adapter.ts

```
@@ -1450,12 +1495,12 @@ export class PostgresUserService implements IServicelocator {
 
         if (customFields) {
           const customFieldAttributes = customFields.reduce(
-            (fieldDetail, { fieldId, fieldAttributes, fieldParams, name }) =>
+            (fieldDetail, { fieldId, fieldAttributes, fieldParams, name, label, type }) =>
               fieldDetail[`${fieldId}`]
                 ? fieldDetail
                 : {
                     ...fieldDetail,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)